atomate/common/firetasks/glue_tasks.py

```diff
@@ -73,6 +73,94 @@ def get_calc_loc(target_name, calc_locs):
         return calc_locs[-1]
 
 
+@explicit_serialize
+class GrabFilesFromCalcLoc(FiretaskBase):
+    """
+    Based on CopyVaspOutputs but for general file copying.
+
+    Required params:
+        filenames (str): filenames to copy.
+        name_prepend (str): string to prepend filenames, e.g. can be a directory.
+        name_append (str): string to append to filenames.
+
+    Optional params:
+        calc_dir: directory to copy from.
```

Comment:
You should indicate in the docs (somewhere) that one of these two optional params should be set.